src/Workspaces/Core/Portable/Utilities/ObjectPools/Extensions.cs

```diff
@@ -3,7 +3,7 @@
 using System.Collections.Generic;
 using System.Text;
 
-namespace Roslyn.Utilities
+namespace Microsoft.CodeAnalysis
```

Comment:
How are we guarding against the potential of a public API ending up in this NuGet?  Since we are using the same namespace as our code base it seems like this could lead to confusing build errors.